cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -132,13 +132,13 @@ pub type SignedExtra = (
 	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
 	BridgeRejectObsoleteHeadersAndMessages,
 	(bridge_to_westend_config::OnBridgeHubRococoRefundBridgeHubWestendMessages,),
-	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
 	frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
+	cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim<Runtime>,
```

Comment:
We had some issues with `StorageWeightReclaim` because it wasn't capturing the entire reclaim. This extension will be changed to an extension that wraps all other extensions to properly capture everything, so placement doesn't really matter here because it will change again. More details in [this PR](https://github.com/paritytech/polkadot-sdk/pull/5234).